fission-cli/library/Fission/CLI/Key/Ed25519.hs

```diff
@@ -16,3 +16,14 @@ parseSecretKey ::
   => ByteArray.ScrubbedBytes
   -> m Ed25519.SecretKey
 parseSecretKey = ensure . parseKey Ed25519.secretKey
+
+-- TODO remove as part of cli-linking
+
+parseKey ::
+     (ByteArray.ScrubbedBytes -> CryptoFailable a)
+  -> ByteArray.ScrubbedBytes
+  -> Either Key.Error a
+parseKey f bytes =
+  case f bytes of
+    CryptoPassed sk  -> Right sk
+    CryptoFailed err -> Left $ Key.ParseError err
```

Comment:
This was removed from `fission-core`. This gets completely cut in the CLI Linking branch. Here to keep thing comping for this PR.